assets/js/views/create.js

```diff
@@ -0,0 +1,38 @@
+var CreateView = Backbone.View.extend({
+  template: AppTemplates.create,
+
+  initialize: function() {
+    this.render();
+
+    // this.listenTo(this.collection, 'sync add', this.render);
+  },
+
+  events: {
+    'submit form': 'createPost'
+  },
+
+  createPost: function(ev) {
+    ev.preventDefault();
+
+    var title = this.$('#title').val();
+    var body = this.$('#body').val();
+
+    this.collection.create({title: title, body: body});
+
+    this.$('#title').val('');
+    this.$('#body').val('');
+
+    this.collection = new Post();
+  },
+
+  render: function() {
+    var html = this.template(this.collection);
+    this.$el.html(html);
+
+    return this;
+  }
+});
+
+var view = new CreateView({
+    el: '#create-target'
```

Comment:
Do not use the `el` property at run time or as part of the constructor, instead manually insert it into your views for better control over which view is being shown and what events are to be fired: http://rtablada.gitbooks.io/front-end-engineering-june-2015/content/backbone/common-views.html#manually-adding-to-the-dom